openrndr-core/src/main/kotlin/org/openrndr/draw/ColorBuffer.kt

```diff
@@ -133,6 +133,7 @@ interface ColorBuffer {
      */
     fun crop(sourceRectangle: IntRectangle) : ColorBuffer {
         val cropped = createEquivalent(width = sourceRectangle.width, height = sourceRectangle.height)
+        cropped.flipV = true
```

Comment:
@edwinRNDR not sure about this.. it didn't work with `cropped.flipV = this.flipV`. Thoughts?